devicon.json

```diff
@@ -8922,6 +8922,24 @@
             }
         ]
     },
+    {
+        "name": "supabase",
+        "altnames": [],
+        "tags": [
+            "cloud-computing-platform",
+            "database",
+            "authentication",
+            "storage"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
```

Comment:
Please add these to the font versions, or create a font version of the icon.
Let us know if you need any help with that. 🙂

Reply:
I was having some trouble while making them so I thought I'd do them after the pull request.

Reply:
I agree with @Panquesito7 :)
If you're really fast you'll get them in the upcoming release